basiclti/basiclti-common/src/java/org/sakaiproject/basiclti/util/SakaiBLTIUtil.java

```diff
@@ -935,10 +935,83 @@ public static void addGlobalData(Site site, Properties props, Properties custom,
 				M_log.debug("Launching with SHA256 Signing");
 			}
 		}
+		
+		//substitute resources params and add the rest
+		if(custom.getProperty(BasicLTIConstants.RESOURCE_LINK_ID) != null){//TODO if allowReviewService prop y future prop for TII
+			addTIIproperties(ltiProps, custom);
```

Comment:
We need to override the resource id, title and description, so we can use the assignment values instead of the site values. That can't be done for each LTI launch within the CR code, that's why it appears here (as out of place as it seems). It would be cool to create some generic methods for special LTI launches like this one, that wouldn't have to rely on TII specific code, but we were in a rush and this was a quick workaround that didn't affect the other LTI calls.